### PR TITLE
ci: use maven plugin from build

### PR DESCRIPTION
**Pull request by raboof**
Cherry-picked from #94

### CODE REVIEW

File: samples/replicatedentity-counter/pom.xml [hunk 1]
```diff
@@ -203,9 +204,9 @@
 
       <!-- FIXME codegen for ReplictedEntity -->
       <plugin>
-        <groupId>com.lightbend</groupId>
-        <artifactId>akkasls-maven-plugin</artifactId>
-        <version>0.31.0</version>
+        <groupId>com.akkaserverless</groupId>
+        <artifactId>akkaserverless-maven-plugin</artifactId>
+	<version>${akkaserverless-maven-plugin.version}</version>
```
Comment (ignasi35) [inline samples/replicatedentity-counter/pom.xml:209]: WhAt is WRong wiTH yOu!?

**Review by ignasi35 — APPROVED** [review]
Tabs over spaces? Seriously?

File: samples/replicatedentity-counter/pom.xml (hunk 1 above)
Comment (ignasi35) [reply samples/replicatedentity-counter/pom.xml:209]: And I think there's more tabs here and there. I just can't look anymore.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**